### PR TITLE
naughty.notify: handle non-existing screen, print a warning

### PR DESCRIPTION
**Pull request by blueyed**
Ref: https://github.com/awesomeWM/awesome/issues/796

### CODE REVIEW

**Comment by Elv13** [comment]
Well, I am not sure this is very user friendly compared to the "old way", but given the problem is already occurring, better merge this.

**Comment by blueyed** [comment]
@awesome-robot 
You're hopefully fixed now.  I've restarted the build for reference: https://travis-ci.org/awesomeWM/awesome/jobs/120732804

**Comment by blueyed** [comment]
> The command "eval git fetch origin +refs/pull/805/merge:" failed. Retrying, 2 of 3.

Meh.

File: lib/naughty/core.lua [hunk 1]
```diff
@@ -437,6 +438,11 @@ function naughty.notify(args)
     local text = args.text or preset.text
     local title = args.title or preset.title
     local s = get_screen(args.screen or preset.screen or screen.focused())
+    if not s then
+        require("gears.debug").print_warning(
+            "naughty.notify: there is no screen available to display the notification.")
```
Comment (psychon) [inline lib/naughty/core.lua]: Should this also print the text (and title?) of the notification?

Comment (blueyed) [inline lib/naughty/core.lua]: Yeah, makes sense after all!

(I've only thought about the `awesome.startup_errors` use case, where it gets printed already.)

**Pull request merged**